### PR TITLE
[Compute AG-KeplerUpdate2] #39872 for CNNF in PCEE

### PR DESCRIPTION
**Pull request by Pubs-MV**
Updates for enabling CNNF on SaaS

### CODE REVIEW

**Review by Pubs-MV — COMMENTED** [review]
more tweaks to content

**Comment by priteshchandaliya** [comment]
LGTM, just a few grammatical errors. Content make sense now and is in correct flow for the reader.

File: compute/admin_guide/firewalls/cnns_saas.adoc [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
+== Cloud Native Network Segmentation (CNNS)
 
-Cloud Native Network Firewall (CNNF) is a Layer 4 network monitoring tool.
-CNNF automatically discovers how entities in your environment communicate, and shows the communication mesh on Radar.
-Radar has a container view, which shows the network topology for your containerized apps.
-Radar also has a host view, which shows the network topology for hosts.
+Cloud Native Network Segmentation (CNNS) is a Layer 4 container-aware virtual firewall and network monitoring tool that enables network segmentation and compartmentalization as a part of a comprehensive defense strategy.
+
+CNNS works as an east-west firewall for containers and hosts. 
+When enabled, CNNS automatically discovers how entities in your environment communicate, and shows the communication mesh on Radar.
```
Comment (alexandre-cezar) [inline compute/admin_guide/firewalls/cnns_saas.adoc]: When enabled, CNNS automatically displays communications in your environment on Radar.
Radar has a container and a host view, where you can view the network topology for your containerized apps and hosts respectively.
You can then define rules to enforce what traffic to allow or block across the network entities. 
It enables you to  identify and block known and unknown traffic moving laterally through your environment.

Comment (Pubs-MV) [reply compute/admin_guide/firewalls/cnns_saas.adoc]: thanks @alexandre-cezar

**Pull request merged**